lib/ui/home/widget/home_survey_page.dart

```diff
@@ -38,15 +36,18 @@ class HomeSurveyPage extends StatelessWidget {
                     Text(
                       survey.title,
                       style: Theme.of(context).textTheme.displayLarge,
-                      maxLines: _textMaxLines,
+                      maxLines: Constants.surveyTitleMaxLines,
                       overflow: TextOverflow.ellipsis,
                     ),
                     const SizedBox(height: AppDimensions.spacing16),
                     Text(
                       survey.description,
                       style: Theme.of(context).textTheme.bodySmall?.copyWith(
-                          color: Colors.white.withOpacity(_descriptionOpacity)),
-                      maxLines: _textMaxLines,
+                            color: Colors.white.withOpacity(
+                              Constants.surveyDescriptionOpacity,
+                            ),
```

Comment:
Can we define a new color from `app_colors` instead? having a const as `surveyDescriptionOpacity` does not make much sense to me 🤔